code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Modifiers.dm

```diff
@@ -0,0 +1,58 @@
+/*
+ * Modifier-applying chemicals.
+ */
+
+/datum/reagent/modapplying
+	name = "brute juice"
+	id = "berserkmed"
+	description = "A liquid that is capable of causing a prolonged state of heightened aggression and durability."
+	taste_description = "metal"
+	reagent_state = LIQUID
+	color = "#ff5555"
+	metabolism = REM
+
+	var/modifier_to_add = /datum/modifier/berserk
+	var/modifier_duration = 2 SECONDS	// How long, per unit dose, will this last?
+
+/datum/reagent/modapplying/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(alien == IS_DIONA)
+		return
+	M.add_modifier(modifier_to_add, dose * modifier_duration)
+
+/datum/reagent/modapplying/cryofluid
+	name = "cryogenic slurry"
+	id = "cryoslurry"
+	description = "An incredibly strange liquid that rapidly absorbs thermal energy from materials it contacts."
+	taste_description = "siberian hellscape"
+	color = "#4CDBDB"
+	metabolism = REM * 0.5
+
+	modifier_to_add = /datum/modifier/cryogelled
+	modifier_duration = 3 SECONDS
+
+/datum/reagent/modapplying/cryofluid/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	..(M, alien, removed)
+	M.bodytemperature -= removed * 20
+
+/datum/reagent/modapplying/cryofluid/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	affect_blood(M, alien, removed * 2.5)
+
+/datum/reagent/modapplying/cryofluid/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
+	affect_blood(M, alien, removed * 0.6)
+
+/datum/reagent/modapplying/cryofluid/touch_mob(var/mob/M, var/amount)
+	if(isliving(M))
+		var/mob/living/L = M
+		for(var/I = 1 to rand(1, round(amount + 1)))
+			L.add_modifier(modifier_to_add, amount * rand(modifier_duration / 2, modifier_duration * 2))
```

Comment:
Due to the nature of RNGesus, it is entirely possible that it will roll all 1.5 seconds, and due to human processing occuring roughly every 2 seconds, that it will spam. From my testing that happened once out of six cycles of 30 units, though any longer risks making it stronger than intended in regards to slowdown buildup.